src/graphcast_agent/mod.rs

```diff
@@ -208,6 +208,8 @@ pub struct GraphcastAgent {
     /// Upon receiving a valid waku signal event of Message type, sender send WakuMessage through mpsc.
     //TODO: currently agent returns the receiver to radio operator, such that radio handler can process WakuMessage however they want. Ideally we should keep WakuMessage within graphcast agent, but for now radio operator is required call generic decode with the specified types. Later we investigate an approach to dynamically register the types during runtime
     pub sender: Arc<SyncMutex<Sender<WakuMessage>>>,
+    /// Keeps track of whether Filter protocol is enabled, if false -> we're using Relay protocol
+    pub filter_protocol_enabled: bool,
```

Comment:
I guess it's not possible to access waku node config settings with the node_handle? can we open an feature request issue for waku please

Reply:
for sure, also it would be good to have the Display trait on the node handle struct so we can debug easier

Reply:
https://github.com/waku-org/waku-rust-bindings/issues/80